examples/perl/app.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env perl
+###############################################################################
+#
+# Simple API for testing support for perl using carton
+#
+###############################################################################
+use strict;
+use warnings;
+use Mojolicious::Lite;
+
+get '/' => { text => 'Hello World' };
```

Comment:
should return "Hello world from tsuru" to make it consistent with other platforms examples